### PR TITLE
Added element manipulation to repos.json

### PR DESCRIPTION
**Pull request by lucasGithuber**
## Description
Added the Element Manipulation addon to repos.json

## Reason
Added it so people can download the addon from the repos page and use auto-updater

### CODE REVIEW

**Review by TheBusyBiscuit — APPROVED** [review]
lgtm

File: resources/repos.json [hunk 1]
```diff
@@ -1412,5 +1412,23 @@
...
+        },
+        "dependencies": {
+            "Minecraft Version(s)": {
+                "1": "1.17.x,1.18.x"
+            },
+            "Slimefun Version": {
+                "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/master/#980\">DEV #980 or above</a>"
+            }
+        }
+    },
```
Comment (TheBusyBiscuit) [inline resources/repos.json]: Good that we have a JSON validator workflow :P

```suggestion
    }
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: resources/repos.json (hunk 1 above)
Comment (ghost) [reply resources/repos.json]: Woopsie

**Comment by lucasGithuber** [comment]
Im not very experienced with github PRs so idk what to do from here 😐

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**